packages/@adobe/spectrum-css-temp/components/calendar/skin.css

```diff
@@ -151,3 +151,53 @@ governing permissions and limitations under the License.
     }
   }
 }
+
```

Comment:
Noticed the following when testing the Calendar/DateRangePicker WHCM. Lemme know if these are as expected due to color limitations or if they aren't/won't be handled in this PR:
1. Can't see that the user is currently focused on the datepicker button:
![focusOnButton](https://user-images.githubusercontent.com/9661127/175695433-7bddb3a2-0ccf-4ee3-a89a-29ff7a403ab8.PNG)
vs
<img width="356" alt="image" src="https://user-images.githubusercontent.com/9661127/175695664-b3d9239a-888a-4608-a0c6-2cd0ede11ab9.png">

2. The current date (aka the 24th) styling is overridden when the range includes it
![currentDayInRange](https://user-images.githubusercontent.com/9661127/175695703-17b6a533-b4fa-44d3-b430-2e7b62692706.PNG)
vs
<img width="427" alt="image" src="https://user-images.githubusercontent.com/9661127/175695780-43b55b8a-83f1-4454-b859-25c606414add.png">

3. The invalid colors are inconsistent between the date range, the validation icons, and the border/focus ring of the input field. The invalid color in the calendar in particular are hard to see on the black blackground (perhaps a limitation of WHCM theming since there isn't an error color?)
![Invalid](https://user-images.githubusercontent.com/9661127/175695843-21ccefe0-1241-446b-a27a-f31a881767f9.PNG)
vs
<img width="438" alt="image" src="https://user-images.githubusercontent.com/9661127/175695882-e3434242-0bf4-41d0-8c8f-546ef897bd4e.png">

Reply:
1. Working on this.
2. I think this is an acceptable compromise. With limited colours knowing today is optional IMO. The colour in the default scheme doesn't meet WCAG so if we insist knowing today is required then we need to fix outside high contrast mode too.
3. These variables were introduced after I created this PR. I'm adding these new invalid variables which will look identical to a selection. The error message is the way you tell that there is an error